### PR TITLE
Docs for Percona Server for MongoDB 0.4.2-3.6.10

### PR DESCRIPTION
**Pull request by timvaillancourt**
## Description
CC: @ryadav88 

## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by timvaillancourt** [comment]
@SarahStegall:

Mesosphere JIRA created here: https://jira.mesosphere.com/browse/DCOS_OSS-4831.

**Comment by timvaillancourt** [comment]
Universe PR: https://github.com/mesosphere/universe/pull/2203

**Comment by ryadav88** [comment]
PR needs review by Docs team but added a `do not merge` label since it needs be merged along with the Universe PR.

**Comment by ryadav88** [comment]
Internal Jira: https://jira.mesosphere.com/browse/DCOS-49951
cc: @SarahStegall @jpe442

**Comment by jpe442** [comment]
@ryadav88 ETA on the Universe PR?

**Comment by ryadav88** [comment]
@jpe442 Universe PR is ready but can be merged once the review here is done. Please ping me or change the label here so we can coordinate both merges together.

File: pages/services/percona-server-mongodb/0.4.2-3.6.10/disaster-recovery/index.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    $ dcos {{ model.serviceName }} backup stop s3
+    ```
+
+## Troubleshooting a backup
+
+To troubleshoot problems with backups, add the flag `--backup-verbose` to the backup command. Please report any issues with `mongodb_consistent_backup` to [https://github.com/Percona-Lab/mongodb_consistent_backup/issues/new](https://github.com/Percona-Lab/mongodb_consistent_backup/issues/new).
+
+# Restoring
+
+Restoring a mongodump-based backup stored on Amazon S3 is possible via the DC/OS web interface and CLI tool, including backups created by the service.
```
Comment (SarahStegall) [inline pages/services/percona-server-mongodb/0.4.2-3.6.10/disaster-recovery/index.md]: Is this supposed to say "mongodump"? Or is that a typo for "mongodb"?

**Comment by SarahStegall** [comment]
I left a few minor changes (spelling, formatting). This is good to go.

**Pull request merged**